app/models/Scholarship.php

```diff
@@ -43,6 +43,25 @@ public static function isClosed($type = 'application')
     return date_has_expired($end_date);
   }
 
+  /**
+   * Determine if the current scholarship application or nomination is open.
+   *
+   * @param  $type  Check dates for either the application or the nomination period.
+   * @return boolean - True if closed, else false.
+   */
+  public static function isOpen($type = 'application')
+  {
+    if ($type === 'application') {
+      $start_date = self::getCurrentScholarship()->application_start;
+    }
+
+    if ($type === 'nomination') {
```

Comment:
I don't think we have a nomination open date, only  scholarship open, so I don't think we need a type for this function